### PR TITLE
CLI slides

### PR DESCRIPTION
**Pull request by decklin**
Did a presentation to cover what's in the command line page of the upstream docs, plus a few more commands, and concepts and history. Probably still too much of an info-dump. Moved the old command line page to a sort of "extra credit" section since it had some unrelated stuff, so we can take whatever needs to be cut for time and move it there. Feedback appreciated! Anyone else is of course welcome to make major edits or run with presenting it.

### CODE REVIEW

File: views/workshop/command_line.deck.markdown [hunk 1]
```diff
@@ -0,0 +1,362 @@
+# Introduction to the command line
+
+Decklin Foster  
+Railsbridge Boston
```
Comment (mxie) [inline views/workshop/command_line.deck.markdown]: `RailsBridge` ;)

File: views/workshop/command_line.deck.markdown [hunk 2]
```diff
@@ -0,0 +1,362 @@
...
+.notes TODO: This is is a link to railsbridge.org! We should probably merge and deploy the updated docs to railsbridgeboston.org, then change the link.
+
+# Getting started
+
+Let's all open up a terminal (command line window) to follow along, if you don't already have one open from installing stuff.
+
+* **On Macs:** In Finder, start Applications > Utilities > Terminal  
+  or find the Terminal application through Spotlight (click the magnifying glass in the top right of the screen and start typing 'Terminal')"
+* **On Windows:** open up Git Shell from your desktop or All Programs menu.
+* **On Linux:** press Ctrl + Alt + T or Find Terminal under the Accessories category of your applications menu.
```
Comment (mxie) [inline views/workshop/command_line.deck.markdown:35]: Perhaps add a line for those using a Chromebook:

```
**On Chromebook:** From the [boxes page](https://www.nitrous.io/app#/boxes), click the "IDE" button. The command line window should appear at the bottom of the screen under the "Console" tab.
```

It looks like this, if you're curious:
nitrous io 2014-09-02 14-26-26 2014-09-02 14-26-58

Comment (mxie) [inline views/workshop/command_line.deck.markdown:35]: I'll update this on the other repo as well.

Comment (decklin) [inline views/workshop/command_line.deck.markdown:35]: Cool, thanks

File: views/workshop/command_line.deck.markdown (hunk 1 above)
Comment (decklin) [inline views/workshop/command_line.deck.markdown]: heh, of course I miss it in the title slide :)

**Comment by decklin** [comment]
updated based on comments.

**Comment by mxie** [comment]
Everything looks great!

To address your concern about the terminal functionality, it's actually just a Linux box that has all the Rails software installed for the user, so I think it'll work fine.

Feel free to rebase and merge when you're ready.

File: views/workshop/command_line.deck.markdown [hunk 3]
```diff
@@ -0,0 +1,363 @@
...
+
+* If you're here, you are capable of picking it up!
+
+# Read Along
+
+Open this web page:
+
+[http://docs.railsbridge.org/ruby/command_line](http://docs.railsbridge.org/ruby/command_line)
+
+.notes TODO: This is is a link to railsbridge.org! We should probably merge and deploy the updated docs to railsbridgeboston.org, then change the link.
```
Comment (mxie) [inline views/workshop/command_line.deck.markdown:26]: FYI, there's a discussion "going on" about separating the command line material as its own curriculum: https://github.com/railsbridge/docs/issues/335

Comment (mxie) [inline views/workshop/command_line.deck.markdown:26]: Also, have you considered submitting these slides over in https://github.com/railsbridge/docs/tree/master/sites/en/workshop?

**Comment by decklin** [comment]
I'll take a look! Merging this now.

**Pull request merged**